.github/workflows/kani.yml

```diff
@@ -24,8 +24,13 @@ jobs:
           cargo build --workspace
 
       - name: Execute Kani regression
+        if: ${{ github.event_name == 'pull_request' }}
         run: ./scripts/kani-regression.sh
 
+      - name: Execute all Kani regressions
+        if: ${{ github.event_name == 'push' }}
+        run: ./scripts/kani-regression.sh all
```

Comment:
What do you think about creating a separate step named Performance Tests instead?

Reply:
Sure. That would work too.